[WD-16470] Page refresh microcloud

goulinkh opened the pull request:
## Done

- Copy update
- Changed Youtube Link
- Added a new quote section
- Update logo assets

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/microcloud
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Make sure that the page matches the design: https://www.figma.com/design/hzQYJM9Tui4wpmyZC0k0WS/23.10-C.com---MicroCloud-Rebranding?node-id=0-1&node-type=canvas&t=gTy0rbPmisXRTtBj-0
- Make sure that the changes in this copy doc are applied: https://docs.google.com/document/d/17iBbFCINLYjPBOYWpgoq9SreyYNkR0Sb5jgx5iTo6MA/edit?tab=t.0


## Issue / Card

Fixes WD-16470

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://canonical-com-1428.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/3567)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

akbarkz commented:
@goulinkh I didn't understand why did the Vanilla patterns appear in this PR? Did you rebase it against the latest main branch?

goulinkh commented:
@akbarkz I've removed the new macros, I did a rebase half way through which didn't show conflicts for macros due to the naming difference `macros`, `_macros`.

The original macro `vf_quoted_wrapper` that I added had some changes to match the design, we may need to make adjustments to the current one `quoted_wrapper` if the designers review it as -1.

mattea-turic commented:
Thanks @goulinkh !

Just a couple of things, even though this was technically only a copy update (but I think it's still worth doing these):

- `<br>` after "is" in "What is a Microcloud?" please
- `<br>` after "scalable" in "fast and scalable cloud infrastructure"
- Muted rule here pls:
<img width="500" alt="Screenshot 2024-11-15 at 09 04 23" src="https://github.com/user-attachments/assets/a35cf682-5b22-4b5f-b246-a16ee2ab893b">

- `<br>` after "and" in the key feature, "Remote authentication, fine grained access control, projects and multi-user setup." of the "Key features" section
- For the three sections: "Edge computing at a large scale", "Scalable cloud infrastructure", and "Efficient dev environment", could the rules above the ctas be muted?
- "Edge computing..." and "Scalable cloud..." should also have regular bottom padding following the cta, like you've done for "Efficient dev environment"
- `<br>` after "laptop" in the paragraph "MicroCloud is lightweight enough that it can run on a developer laptop. You can use it for:" in the section "Efficient dev environment"
- I believe the tag where in the header where "Microcloud" sits in the nav should maybe be the product tag, but I'd like to verify this with @Sophie-32 or @akbarkz

goulinkh commented:
Thanks for the review @mattea-turic I've applied the suggested changes (except the last point).

mattea-turic commented:
Looks great, thank you @goulinkh 

I'll add a +1 for design – and we'll see what the outcome of the product tag will be once the others get back to me

Sophie-32 commented:
missing a copy edit in this section - 
![Screen Shot 2024-11-15 at 11 05 44 AM](https://github.com/user-attachments/assets/4c81f2c9-8a78-4f90-abe1-a9afb06bfba9)

goulinkh commented:
Thanks for reporting this @Sophie-32, I've fix the spelling to US.

lyubomir-popov commented:
This copy is a lot - can we please switch to  h6 instead of h4? This will take it down to paragraph fonti-size, and reduce scrolling

mattea-turic commented:
@goulinkh Update on the tag - we should be using the tag that comes with the product logo, so in this case it would be the one with the cloud. I'm not sure how replacing this works on the dev side - do you know where the asset would be for this or should I provide you with a file of a specific format?

goulinkh commented:
I've made the changes could you review the changes based on your feedback please?

mattea-turic commented:
@goulinkh Looks good, thanks (and appreciate how responsive you are with these changes)

akbarkz commented:
@goulinkh there was one small error from djLint reported in CI, phal

akbarkz commented on templates/microcloud/index.html:
```diff
@@ -1,405 +1,514 @@
+        </ul>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="https://canonical-microcloud.readthedocs-hosted.com/en/stable/tutorial/">Deploy your own MicroCloud following this tutorial&nbsp;&rsaquo;</a>
```
For me this link did not open, I got 404. I tagged the stakeholder in the copy doc about this, not a blocker.
